.github/workflows/call-run-tests.yml

```diff
@@ -14,14 +14,20 @@ jobs:
           node-version: 20
           cache: 'npm'
 
+      - name: Install jq
+        run: sudo apt-get install jq -y
+
       - name: Install dependencies
         run: npm ci
 
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps
 
-      - name: Run Playwright tests
-        run: ./run-tests.sh
+      - name: Run Playwright short tests
```

Comment:
general suggestion: Please use a short yaml comment to describe for reviewers and future maintainers any terms that are "special" -- that is not immediately clear what they mean.

Looking at `run-tests.sh` right now, I can see the separation. But I don't have a clear sense of "why" this separation is being made.
Now I can guess that "long" means "longer running."
But I can't guess many other important things: 
- why it's being done
- what the next developer should do when they add tests or a "short" test gets slower to run